### ISSUE TITLE
Manifest.json not being injected

### ISSUE THREAD

**Issue by jlanzarotti** [quasarframework/quasar-cli#149]
Hi! 

Using the CLI and configuring everything for pwa (following the current guideline), it does not inject the manifest.json link.

However, if I edit the index.template.html and put ```<link rel="manifest" href="/manifest.json">```  it is showed in the chrome inspect -> Application after a reload. This line it is supposed to be auto-injected correct ? 

Running with: 

```
quasar dev -m pwa
```

quasar info

```
Operating System        	Darwin(17.7.0) - darwin/x64
NodeJs                  	8.11.1

Global packages
  NPM                   	6.2.0
  yarn                  	Not installed
  quasar-cli            	0.16.4
  vue-cli               	3.0.0-rc.3
  cordova               	Not installed

Important local packages
  quasar-cli            	0.16.4	(Quasar Framework CLI)
  quasar-framework      	0.16.0	(Build responsive websites, PWAs, hybrid mobile apps and Electron apps, all simultaneously using same codebase)
  quasar-extras         	2.0.2	(Quasar Framework fonts, icons and i18n.)
  vue                   	2.5.16	(Reactive, component-oriented view layer for modern web interfaces.)
  vue-router            	3.0.1	(Official router for Vue.js 2)
  vuex                  	3.0.1	(state management for Vue.js)
  electron              	Not installed
  electron-packager     	Not installed
  electron-builder      	Not installed
  @babel/core           	7.0.0-beta.49	(Babel compiler core.)
  webpack               	4.9.1	(Packs CommonJs/AMD modules for the browser. Allows to split your codebase into multiple bundles, which can be loaded on demand. Support loaders to preprocess files, i.e. json, jsx, es7, css, less, ... and your custom stuff.)
  webpack-dev-server    	3.1.4	(Serves a webpack app. Updates the browser on changes.)
  workbox-webpack-plugin	3.2.0	(A plugin for your Webpack build process, helping you generate a manifest of local files that workbox-sw should precache.)

```

**Comment by rstoenescu** [comment]
Hi,

Already fixed in v0.17 beta (releasing the official version today or tomorrow). Furthermore, there's a big enhancement coming up, which is that you can play with the service worker in dev too! Finally found a way to avoid the HMR issue.

**Comment by jlanzarotti** [comment]
Amazing!! Thanks for the hard work!
